packages/main/src/RPA/Salesforce.py

```diff
@@ -163,10 +201,13 @@ class Salesforce:
 
     account = {"Name": None, "Id": None}
 
-    def __init__(self, sandbox: bool = False) -> None:
+    def __init__(self, sandbox: bool = False, domain: str = "login") -> None:
         self.logger = logging.getLogger(__name__)
         self.sf = None
-        self.domain = "test" if sandbox else "login"
```

Comment:
Optional nit:
`if sandbox and domain != "login": raise SalesforceDomainChangeError("Can't use a custom domain with sandbox on")`

Feel free to change the error type/message and use a constant like `DEFAULT_DOMAIN = "login"`. Or not apply this suggestion if you feel that's breaking or too intrusive.

Reply:
Based on the conditional in the __init__ function I believe that setting sandbox = true will override the custom domain. So this may be unnecessary. Correct me if I missed something.

Reply:
Correct! But what if the user provides both by mistake and doesn't know what he/she's doing? We can help him/her with an error :D.

Since you merged and this is not as important anymore, we can leave it flexible like it is.